packages/common/src/utils/common.ts

```diff
@@ -6,6 +6,12 @@ export function debounce<T extends (...args: any[]) => void>(callback: T, wait =
   };
 }
 
+export const unicodeToChar = (text: string) => {
+  return text.replace(/\\u[\dA-F]{4}/gi, (match) => {
+    return String.fromCharCode(parseInt(match.replace(/\\u/g, ''), 16));
+  });
+};
```

Comment:
This was not needed before, what is the reason it is now?

Reply:
It renders` \u00a9` literally in the footer. I tried escaping it like` \\u00a9` and some other definitions approaches without succes. After about 15m i resorted to this approach.